### PR TITLE
Clean up constants

### PR DESCRIPTION
**Pull request by alanna-zhou**
Centralized all the constants in one file, and removed the postman token because it was useless.

Again, sorry for the duplicate PR--for some reason, using the RETRY_COUNT constant in RequestUtils.js broke everything. Will be deleting the original PR once this gets merged in.

Currently testing on dev server (transit-dev:alanna) because it sometimes works and sometimes doesn't work

### CODE REVIEW

File: src/utils/Constants.js [hunk 1]
```diff
@@ -1,11 +1,91 @@
...
+const WALK_SPEED = 3.0;
+
+/* Degrees Precision constants */
+const DEG_EQ_PRECISION = 5; // 5 degrees of precision is about a 1.1 meters, is a stop
+const DEG_EXACT_PRECISION = 6; // 6 degrees of precision is about a 111 mm, is exact point
+const DEG_KM_PRECISION = 2; // 3 degrees of precision is about 1 km, stop barely walkable
+const DEG_MAX_PRECISION = 6;
+const DEG_MIN_PRECISION = 1;
+const DEG_NEARBY_PRECISION = 4; // 4 degrees of precision is about 11 meters, stop nearby
+const DEG_WALK_PRECISION = 3; // 3 degrees of precision is about 111 meters, stop walkable
```
Comment (young-k) [inline src/utils/Constants.js]: I think we should actually not alphabetize and have these in order, it'd be clearer for the future

File: src/utils/Constants.js [hunk 2]
```diff
@@ -1,11 +1,91 @@
...
+/* Delay Buffer constants */
+// buffer to account for routes in past 20 minutes with delays
+const FIRST_DELAY_BUFFER_IN_MINUTES = 20;
+// additional buffer to account for time needed to walk from current location to bus stop
+const SECOND_DELAY_BUFFER_IN_MINUTES = 40;
+
+/* Distance & Speed constants */
+// > 3.0 suggests getting off bus earlier and walk half a mile instead of waiting longer
+const MAX_WALK_DIST_PER_LEG = 2000;
+const AUTOCOMPLETE_RADIUS = 24140;
```
Comment (young-k) [inline src/utils/Constants.js]: we probably need a comment here, if you're not sure could you add a TODO here?

File: src/utils/Constants.js [hunk 3]
```diff
@@ -1,11 +1,91 @@
...
+  max: 1000, // max 1000 routes in storage
+  maxAge: 1000 * 60 * 15, // max age 15 minutes
+};
+const QUERY_PREDICTIONS_CACHE_OPTIONS = {
+  max: 10000, // Maximum size of cache
+  maxAge: 1000 * 60 * 60 * 24 * 5, // Maximum age in milliseconds
+};
+
+/* Count constants */
+const MIN_FUZZ_RATIO = 75;
```
Comment (young-k) [inline src/utils/Constants.js]: what is a fuzz ratio? probably useful to put that here

File: src/utils/Constants.js [hunk 4]
```diff
@@ -1,11 +1,91 @@
...
+  max: 10000,
+  maxAge: 1000 * 60 * 60 * 24 * 5,
+};
+const ROUTES_CALC_CACHE_OPTIONS = {
+  max: 1000, // max 1000 routes in storage
+  maxAge: 1000 * 60 * 15, // max age 15 minutes
+};
+const QUERY_PREDICTIONS_CACHE_OPTIONS = {
+  max: 10000, // Maximum size of cache
+  maxAge: 1000 * 60 * 60 * 24 * 5, // Maximum age in milliseconds
```
Comment (young-k) [inline src/utils/Constants.js]: for the middle `maxAge` we put a comment with the number of minutes, can we do something similar for the first one and this one as well?

File: src/utils/AllStopUtils.js [hunk 5]
```diff
@@ -81,8 +72,9 @@ function fetchPrecisionMaps() {
  * @param degreesPrecision
  * @returns {Promise<void>}
  */
-async function getPrecisionMap(degreesPrecision: ?number = DEG_EQ_PRECISION) {
-  if (degreesPrecision < 1 || degreesPrecision > 6) {
+async function getPrecisionMap(degreesPrecision: ?number = Constants.DEG_EQ_PRECISION) {
+  if (degreesPrecision < Constants.DEG_MIN_PRECISION
+    || degreesPrecision > Constants.DEG_MAX_PRECISION) {
```
Comment (kq-li) [inline src/utils/AllStopUtils.js]:
```
if (
  degreesPrecision < Constants.DEG_MIN_PRECISION
  || degreesPrecision > Constants.DEG_MAX_PRECISION
) {
```

File: src/utils/Constants.js (hunk 1 above)
Comment (kq-li) [reply src/utils/Constants.js]: these don't really seem to reference the same value, how would you order them?

Comment (young-k) [reply src/utils/Constants.js]: oh i meant in terms of like 1-6 in terms of value, but I don't have a strong preference!

Comment (kq-li) [reply src/utils/Constants.js]: oh duh LOL me neither

File: src/utils/Constants.js [hunk 6]
```diff
@@ -24,17 +25,18 @@ const SECOND_DELAY_BUFFER_IN_MINUTES = 40;
 /* Distance & Speed constants */
 // > 3.0 suggests getting off bus earlier and walk half a mile instead of waiting longer
 const MAX_WALK_DIST_PER_LEG = 2000;
+// The distance (in meters) within which to return Google place results for autocomplete.
 const AUTOCOMPLETE_RADIUS = 24140;
 const WALK_SPEED = 3.0;
 
 /* Degrees Precision constants */
+const DEG_MIN_PRECISION = 1;
+const DEG_KM_PRECISION = 2; // 3 degrees of precision is about 1 km, stop barely walkable
```
Comment (kq-li) [inline src/utils/Constants.js]: should this comment say 2 degrees?

File: src/utils/Constants.js [hunk 7]
```diff
@@ -24,17 +25,18 @@ const SECOND_DELAY_BUFFER_IN_MINUTES = 40;
...
+const DEG_WALK_PRECISION = 3; // 3 degrees of precision is about 111 meters, stop walkable
+const DEG_NEARBY_PRECISION = 4; // 4 degrees of precision is about 11 meters, stop nearby
 const DEG_EQ_PRECISION = 5; // 5 degrees of precision is about a 1.1 meters, is a stop
```
Comment (kq-li) [inline src/utils/Constants.js]: `about a 1.1` => `about 1.1`

File: src/utils/Constants.js [hunk 8]
```diff
@@ -24,17 +25,18 @@ const SECOND_DELAY_BUFFER_IN_MINUTES = 40;
...
 const DEG_EXACT_PRECISION = 6; // 6 degrees of precision is about a 111 mm, is exact point
```
Comment (kq-li) [inline src/utils/Constants.js]: `about a 111` => `about 111`

File: src/utils/AllStopUtils.js [hunk 9]
```diff
@@ -74,7 +74,8 @@ function fetchPrecisionMaps() {
  */
 async function getPrecisionMap(degreesPrecision: ?number = Constants.DEG_EQ_PRECISION) {
   if (degreesPrecision < Constants.DEG_MIN_PRECISION
```
Comment (kq-li) [inline src/utils/AllStopUtils.js]: newline after the initial `if (` is cleaner imo but your call

Comment (young-k) [reply src/utils/AllStopUtils.js]: I agree with @kq-li here!

File: src/utils/Constants.js [hunk 10]
```diff
@@ -1,12 +1,17 @@
...
   max: 10000, // Max 10000 autocomplete results
   maxAge: 1000 * 60 * 60 * 24 * 5, // Max age in 5 days
 };
+
+// A cache containing saved routes IDs to routes
 const ROUTES_CALC_CACHE_OPTIONS = {
   max: 1000, // Max 1000 routes
   maxAge: 1000 * 60 * 15, // Max age in 15 minutes
 };
+// A cache mapping a /search's request query string to its Google autocomplete predictions
```
Comment (kq-li) [inline src/utils/Constants.js]: this newline spacing is inconsistent with the above change

**Pull request merged**